Tests/SwiftDocCTests/Rendering/RenderBlockContent+AsideTests.swift

```diff
@@ -0,0 +1,463 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright (c) 2025 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See https://swift.org/LICENSE.txt for license information
+ See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+
+import Foundation
+import Markdown
+@testable import SwiftDocC
+import Testing
+
+struct RenderBlockContent_AsideTests {
+
+    typealias Aside = RenderBlockContent.Aside
+    typealias AsideStyle = RenderBlockContent.AsideStyle
+
+    let testBlock: RenderBlockContent = .paragraph(
+        RenderBlockContent.Paragraph(
+            inlineContent: [
+                RenderInlineContent.text("This is a test paragraph")
+            ]
+        )
+    )
+
+    private func testStyle(for name: String) -> AsideStyle {
+        .init(rawValue: name)
+    }
+
+    private func decodeAsideRenderBlock(_ json: String) throws -> Aside? {
+        guard let data = json.data(using: .utf8) else {
+            Issue.record("Found unexpected string encoding.")
+            return nil
+        }
+        let decodedBlock = try JSONDecoder().decode(RenderBlockContent.self, from: data)
```

Comment:
FYI: A string's UTF8 data can always be turned into a `Data` value but that is not representable in the `data(using:)` function because _that_ can accept any encoding and some of those encodings can return nil. 

If you use `Data(json.utf8)` instead you don't need the first half of this helper function. 

```suggestion
        let decodedBlock = try JSONDecoder().decode(RenderBlockContent.self, from: Data(json.utf8))
```